### PR TITLE
chore: Update Windows base image version to 17763.1637.2012040632

### PR DESCRIPTION
**Pull request by AbelHu**
chore: Update Windows base image version to 17763.1637.2012040632

### CODE REVIEW

**Comment by AbelHu** [comment]
@mainred All cases pass.

**Comment by mainred** [comment]
Sorry, the passed test does not answer my question. Because we have no patch check in the VHD test.

File: vhdbuilder/packer/windows-image.env [hunk 1]
```diff
@@ -4,6 +4,6 @@
 # CLI example to get the latest image version:
 #   az vm image show --urn MicrosoftWindowsServer:WindowsServer:2019-Datacenter-Core-with-Containers-smalldisk:latest
 WINDOWS_2019_BASE_IMAGE_SKU=2019-Datacenter-Core-smalldisk
-WINDOWS_2019_BASE_IMAGE_VERSION=17763.1579.2011180054
+WINDOWS_2019_BASE_IMAGE_VERSION=17763.1637.2012040632
```
[the remark this answers is not in the document]
Comment (AbelHu) [reply vhdbuilder/packer/windows-image.env:7]: yes

**Review by mainred — APPROVED** [review]

File: vhdbuilder/packer/windows-image.env (hunk 1 above)
[the remark this answers is not in the document]
Comment (mainred) [reply vhdbuilder/packer/windows-image.env:7]: Before removing 12B patch, building vhd report the following error:
Error during install of windows10.0-kb4592440-x64_e4897b096fb33c7608a18b54d6e9b0a142536683.msu. ExitCode: 2359302
2359302 is explained as `The update to be installed is already installed on the system.` which means we have 12B in `17763.1637.2012040632` already.

**Pull request merged**